firmware/config/boards/hellen/hellen72/board.mk

```diff
@@ -6,19 +6,15 @@ BOARDINC = $(BOARDS_DIR)/hellen/hellen72
 
 # Target processor details
 ifeq ($(PROJECT_CPU),ARCH_STM32F4)
-  MCU_DEFS  = -DSTM32F427xx
```

Comment:
If you remove 427 define from here, then where is it now? We cannot use 405 defs for 427 and 469, these are different chips with their own ChibiOS/os/common/ext/ST/STM32F4xx/* includes and stm32f4xx_hal_flash_ex.* support.

Reply:
F405/7 and F427/9 have the same linker map.  The F427/9 have more RAM, but are otherwise the same layout. I've been running an F405 linker config on an F427 for a year without issue.

The flash controller is actually different, but the F42x and F46x flash controllers are fully backward compatible with the F40x.  The only difference is that the F42x _can_ be dual bank on high density devices, but we don't currently support it.  We always put the config in the same spot for all F4 chips.  As mentioned, many MRE and Proteus boards are running what's technically an F405/7 image on an F427/9.

Reply:
The whole idea of using 427 instead of 407 is for its benefits like more RAM and the 2nd flash bank (which we really need for config saving while the engine is running). Even if we are not using them now, we are definitely going to (otherwise we would solder 407s all the time because they are just cheaper and more in stock). Therefore, we need a way to distinguish between different STM32F4 families. Therefore, we need STM32F*** defines.

Reply:
There is plenty of more implementation needed before we can use dual bank at all on the F4 parts (changes in mpu_util, for example)   The purpose of this change is to start reducing the duplication, moving stuff in to hw_ports.mk instead of in board.mk.  When we need specific support for the dual bank devices, this will make it easier to make that change.

I don't want to add explicit 42x/469 support yet because:
- we don't actually use the added functionality yet
- it will add additional work while there is lots of churn happening, while simply adding it later will be much less work.